File: CompanyAPP/ClientApp/src/app/models/customer.ts
import { Note } from "./note";
import { Status } from "./status";

export class Customer {
  pkCustomerId: number;
  firstname: string;
  lastname: string;
  email: string;
  cellNo: string;
  address: string;
  city: string;
  region: string;
  country: string;
  postalCode: string;
  customerId: string;
  status: string;
  createDate: string;
  passportNo: string;
  fkStatusId: number;
  fkStatus: Status;
  note: Note[];
}

File: CompanyAPP/Models/Customer.cs
using System;
using System.Collections.Generic;

namespace CompanyAPP.Models
{
    public partial class Customer
    {
        public Customer()
        {
            Note = new HashSet<Note>();
        }

        public int PkCustomerId { get; set; }
        public string Firstname { get; set; }
        public string Lastname { get; set; }
        public string Email { get; set; }
        public string CellNo { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string Region { get; set; }
        public string Country { get; set; }
        public string PostalCode { get; set; }
        public Guid? CustomerId { get; set; }
        public string Status { get; set; }
        public string CreateDate { get; set; }
        public string PassportNo { get; set; }
        public int? FkStatusId { get; set; }

        public Status FkStatus { get; set; }
        public ICollection<Note> Note { get; set; }
    }
}
